test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionExecutorTests.cs

```diff
@@ -1,509 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
But why? Why can't the functionality on `ObjectMethodExecutor` be tested directly anymore?